test/integration/apiserver/tracing/tracing_test.go

```diff
@@ -208,7 +208,7 @@ func TestAPIServerTracingWithEgressSelector(t *testing.T) {
 	defer os.Remove(egressSelectorConfigFile.Name())
 
 	if err := os.WriteFile(egressSelectorConfigFile.Name(), []byte(`
-apiVersion: apiserver.config.k8s.io/v1beta1
```

Comment:
`apiserver.config.k8s.io/v1beta1` was never a valid Group/Version for `EgressSelectorConfiguration` (the correct Group/Version is `apiserver.k8s.io/v1beta1`), but now we are correctly (and strictly) validating the config.

Reply:
Good catch